pkgs/applications/misc/snowsql/linux.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  rpmextract,
+  makeWrapper,
+  openssl,
+  libxcrypt-legacy,
+  zlib,
+  version,
+  pname,
+  meta,
+}:
+stdenv.mkDerivation rec {
```

There is a preference to use finalAttrs rather than rec. Note that the newly opened parenthesis needs to be closed at the end of the package (see suggestion).
```suggestion
stdenv.mkDerivation (finalAttrs: {
```